src/bucket.ts

```diff
@@ -2874,6 +2875,47 @@ class Bucket extends ServiceObject {
     );
   }
 
+
+  setCorsConfiguration(corsConfiguration: Cors[]): Promise<SetBucketMetadataResponse>;
+  setCorsConfiguration(
+    corsConfiguration: Cors[],
+    callback: SetBucketMetadataCallback
+  ): void;  
+  /**
+   * This can be used to set the CORS configuration on the bucket. 
+   * 
+   * The configuration will be overwriteten with the value passed into this.
+   *
+   * @param {Cors[]} corsConfiguration The new CORS configuration to set
+   * @param {SetBucketMetadataCallback} [callback] Callback function.
+   * @returns {Promise<SetBucketMetadataResponse>}
+   *
+   * @example
+   * const storage = require('@google-cloud/storage')();
+   * const bucket = storage.bucket('albums');
+   *
+   * const corsConfiguration = [{"maxAgeSeconds": 3600}]; // 1 hour
```

Comment:
You can lose the quotation marks around `maxAgeSeconds` :)

Reply:
:+1: